volumes.xconf

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <collection xmlns="http://exist-db.org/collection-config/1.0">
+    <triggers>
+        <trigger event="update, create" class="org.exist.collections.triggers.XQueryTrigger">
```

Comment:
The `collection.xconf.xsd` schema seems to steer us toward creating one `<trigger>` per event type: https://github.com/eXist-db/exist/blob/develop/schema/collection.xconf.xsd#L385-L397.